src/main/kotlin/uk/gov/justice/digital/hmpps/prisonercontactregistry/client/PrisonApiClient.kt

```diff
@@ -24,7 +24,7 @@ class PrisonApiClient(
 
   fun getOffenderContacts(offenderNo: String): ContactsDto? {
     return webClient.get()
-      .uri("/api/offenders/$offenderNo/contacts?approvedVisitorsOnly=true&activeOnly=true")
+      .uri("/api/offenders/$offenderNo/contacts?approvedVisitorsOnly=true")
```

Comment:
Sorry guys wondering if we should remove this also : 
?approvedVisitorsOnly=true
As https://developer-portal.hmpps.service.justice.gov.uk/components/prisoner-content-hub-frontend replies on the code being as it was the other week.

Reply:
Oops - merged before seeing this comment. I guess we want the ability to set `approvedVisitorsOnly` so perhaps just add this as an optional (defaulting to `false`) filter on the contact registry API?